skel/flatpak/overrides/global

```diff
@@ -0,0 +1,6 @@
+[Environment]
+QT_LINUX_ACCESSIBILITY_ALWAYS=true
```

Comment:
This should have been `QT_LINUX_ACCESSIBILITY_ALWAYS=1` but it is not required as the documentation stands:
> In the Unix/X11 AT-SPI implementation, applications become accessible when two conditions are met:
>
>    org.a11y.Status.IsEnabled DBus property is true
>    org.a11y.Status.ScreenReaderEnabled DBus property is true
>
>An alternative to setting the DBus AT-SPI properties is to set the QT_LINUX_ACCESSIBILITY_ALWAYS_ON environment variable.

So the `org.a11y.bus=talk` should be enough